tests/test_views.py

```diff
@@ -687,43 +499,31 @@ def test_view_aliases_using_site_filter(self):
             language='en',
             body='Das ist Text auf Deutsch',
         )
-        site1_alias = self._create_alias([site1_plugin], site=site1, name='site1_alias', category=self.category)
-        site2_alias = self._create_alias([site1_plugin], site=site2, name='site2_alias', category=self.category)
-        alias_content_site1 = AliasContent.objects.create(
```

Comment:
What was the reasoning behind removing the content creation?

Reply:
It was removed to keep the test DRY as I noticed that the [_create_alias method](https://github.com/django-cms/djangocms-alias/blob/277d0b1c06740dfa10c4ed161af34e83d6b6de81/tests/base.py#L88) handled the creation of `AliasContent`, as well as creating a `Version` and populating the plugins.

Reply:
Looking further into it it does appear to be making the same content although be careful when doing this in future because some tests create new content on purpose to ensure that the first content is version 1 = Published and then next new content would be version 2 = Draft. The tests could still pass but they will then no longer be protecting against any multiple versioning filtering.